xmtp/Cargo.toml

```diff
@@ -43,4 +43,4 @@ url = "2.3.1"
 
 [dependencies.libsqlite3-sys]
 version = "0.26.0"
-features = ["bundled-sqlcipher"]
+features = ["bundled-sqlcipher-vendored-openssl"]
```

Comment:
Fixes https://github.com/xmtp/libxmtp/actions/runs/5165745304/jobs/9305449137

Reply:
This features forces libsqlite to statically build a copy of openssl into the project, however will cause version mismatch issues when other crates are using a non vendored version.